app/controllers/api/v2/monetary_exchange_rates_controller.rb

```diff
@@ -2,7 +2,7 @@ module Api
   module V2
     class MonetaryExchangeRatesController < ApiController
       def index
-        jan_five_back = 5.years.ago.change({ day: 01, month: 01, hour: 0, minute: 0, second: 0 })
+        jan_five_back = 5.years.ago.change({ day: 0o1, month: 0o1, hour: 0, minute: 0, second: 0 })
```

Comment:
```suggestion
        jan_five_back = 5.years.ago.change({ day: 1, month: 1, hour: 0, minute: 0, second: 0 })
```

Reply:
Yeah this is very annoying when auto correct does this...